patch_recurrence/steps.py

```diff
@@ -22,20 +44,19 @@ def scale(img, scales):
 
 def generate_patches(scaled_imgs, constants, all_patches):
     patch_size = constants.PATCH_SIZE
-    std_dev_threshold = constants.STD_DEV_THRESHOLD
+    step = 1 if all_patches else 2
```

Comment:
Nope. The reason I did this is because I am considering alternate patches of all scaled images initially (all_patches = 0). Later for the t-map estimation, I require all patches of just the original image (all_patches = 1). I will try to see if there is a better way to do this.